p2p/peer.py

```diff
@@ -864,6 +865,8 @@ def _add_peer(self,
             peer.add_subscriber(subscriber)
             for msg in msgs:
                 subscriber.add_msg(msg)
+        if self.event_bus is not None:
+            self.event_bus.broadcast(PeerConnectedEvent(peer))
```

Comment:
I'm not sure how often `PeerConnnectedEvent` occurs in normal networking traffic but let me just raise awareness that broadcasting very frequent events via the event bus may cause performance issues. There's a lot that we can optimize about it (see, https://github.com/ethereum/lahja/issues/2 and https://github.com/ethereum/lahja/issues/1) but as of today, every event broadcasted travels at least one hop from the process that it originates from (`networking` in this case) to the `main` process where the central event bus coordinator lives and then from there to each and every other connected process (unless `BroadcastConfig(filter_endpoint='some_specific_endpoint')` was used).

Reply:
It shouldn't be frequent at all. Typically once every several seconds. Although it's possible that some certain corner case might cause peers to drop and reconnect a lot in a short period. I wouldn't expect it to be more than a burst.